lib/Inline/Python.pm6

```diff
@@ -542,7 +542,12 @@ BEGIN {
     PythonObject.^compose;
 }
 
-multi sub EVAL(Cool $code, Str :$lang where { ($lang // '') eq 'Python' }, PseudoStash :$context) is export {
-    state $py = ::("Inline::Python").default_python;
-    $py.run($code);
+multi sub EVAL(
+        Str $code,
```

Comment:
mmm. Probably not THAT intentional, just seemed better. I'll undo to go with consistency.